README.rst

```diff
@@ -1,8 +1,8 @@
 Introduction
 ============
 
-.. image:: https://readthedocs.org/projects/adafruit-circuitpython-tinylora/badge/?version=latest
-    :target: https://circuitpython.readthedocs.io/projects/tinylora/en/latest/
+.. image:: https://readthedocs.org/projects/circuitpython-tinylora/badge/?version=latest
+    :target: https://circuitpython-tinylora.readthedocs.io/en/latest/
```

Comment:
Why did you change this? If the URL provided by cookiecutter isn't working it means there's an issue on the RTD end that needs to be addressed. TinyLoRa documentation doesn't appear to have been setup properly in RTD. Please ping me for the proper process for this.